openapi_python_client/parser/properties/__init__.py

```diff
@@ -268,9 +268,6 @@ def build_model_property(
 
     for key, value in all_props.items():
         prop_required = key in required_set
-        if not isinstance(value, oai.Reference) and value.allOf:
```

Comment:
Hmm.. I remember this being important for some reason..
I think removing this might inlines all references rather than referencing them as objects.
You might have more context at this point though.

Reply:
@dtkav Can you clarify what you mean that it might inline all references?

Reply:
I generated the benchling client with both branches, and I think this is actually fine.
I was just trying (but failing) to remember the context around the "resolved later" comment.

It look like your PR generates 40 additional models which seem to correspond to actual missing functionality. 👍

Reply:
There was a refactor upstream which I adapted the initial `allOf` implementation for. I think I shouldn't have included this there.
In that refactor, he changed the way `property_from_data` works such that it resolves references to models it has already built instead of returning a `RefProperty` that gets resolved later.